Maple2.Model/Game/FieldInstance.cs

```diff
@@ -1,17 +1,26 @@
 ﻿using System.Runtime.InteropServices;
 using Maple2.Model.Enum;
+using Maple2.PacketLib.Tools;
+using Maple2.Tools;
 
 namespace Maple2.Model.Game;
 
-[StructLayout(LayoutKind.Sequential, Size = 6)]
-public readonly struct FieldInstance {
+public sealed class FieldInstance : IByteSerializable {
     public readonly bool BlockChangeChannel;
     public readonly InstanceType InstanceType;
     public readonly int InstanceId;
 
+    public FieldInstance() { }
+
```

Comment:
tbh not a fan of having empty constructors. Can we do something similar to how BeautyShopCost has a static Zero?

Reply:
fixed